workshop/exercise-3/README.md

```diff
@@ -1,79 +1,64 @@
-## Setup: Install Istio and Knative on Your Cluster
+## Deploy our First Application to Knative using the Knative Client (kn)
+A relatively new project, the knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple node.js with express app which returns the first n numbers of the Fibonacci sequence. To use the app, deploy it, and simply make a GET request to the `/` endpoint with a number as the parameter. 
```

Comment:
The last sentence sounds a bit awkward to me, how about:  Once deployed, to use the app simply make...

Reply:
Agree - updated.